docs/reference/setup/install/systemd.asciidoc

```diff
@@ -21,6 +21,17 @@ These commands provide no feedback as to whether Elasticsearch was started
 successfully or not. Instead, this information will be written in the log
 files located in `/var/log/elasticsearch/`.
 
+If you have password-protected your {es} keystore, you will need to provide
+`systemd` with the keystore password using a local file and systemd environment
```

Comment:
> ... you will need to provide .. the keystore password using a local file ...

Do we need to provide a tip about protecting that local file since it contains the password?